kedro-datasets/pyproject.toml

```diff
@@ -165,7 +165,11 @@ yaml-yamldataset = ["kedro-datasets[pandas-base]", "PyYAML>=4.2, <7.0"]
 yaml = ["kedro-datasets[yaml-yamldataset]"]
 
 # Experimental Datasets
-
+langchain-chatopenaidataset = ["langchain-openai"]
```

Comment:
Shall we maybe fix at least the major versions as langchain API is changing so fast?

Reply:
Done!